cron/backup.conf

```diff
@@ -7,3 +7,8 @@ backup_dobackup = 1,1,1,1,1,1,1,1
 backup_rotation = none,none,monthly,daily,none,none,none,none
 backup_pruning = 0,0,0,180,0,0,0,0
 weekly_rotation_indices = events
+refresh_interval = 1s
```

Comment:
This should be set to 30s 
With as much indexing as we do setting it to 1s would create way too much overhead for our cluster.

Reply:
For our specific config, I believe this is the case. Since this is the default, I used that value here.

Reply:
ahh okies :) ty